src/app/shared/url-resolver.service.ts

```diff
@@ -46,6 +46,9 @@ export class UrlResolverService {
     title = this.decodedString(title.replace(`/containers/`, ''));
     title = this.decodedString(title.replace(`/my-notebooks/`, ''));
     title = this.decodedString(title.replace(`/notebooks/`, ''));
+    title = this.decodedString(title.replace(`/search-workflows/`, ''));
+    title = this.decodedString(title.replace(`/search-containers/`, ''));
+    title = this.decodedString(title.replace(`/apptools/`, ''));
```

Comment:
Just going off the function name, it seems like these three shouldn't be here because the URLs don't have the entry paths. For example, clicking on a workflow from `/search-workflows` takes you to `/workflows/<entry-path>`, not `/search-workflows/<entry-path>`

Why are those three needed?

Reply:
I guess if the path happened to be `/search-workflows/<entry-path>` then it would redirect to the correct workflow at `/workflows/<entry-path>`. But you're right, that's not the purpose of those three routes so I think I'll remove those lines.